apps/andi/lib/andi_web/validators/dataset_schema_validator.ex

```diff
@@ -0,0 +1,53 @@
+defmodule AndiWeb.DatasetSchemaValidator do
+  @moduledoc """
+  Used to validate dataset schemas
+  """
+
+  def validate(%{"technical" => %{"sourceType" => source_type}} = dataset) when source_type in ["ingest", "stream"] do
+    SimplyValidate.validate(dataset, [validate_schema_required()]) ++ validate_schema(dataset)
+  end
+
+  def validate(_), do: []
```

Comment:
So if a definition isn't ingest or stream, or has no source type or technical, we just mark it valid? I'm guessing because xml is the first time we actually validate a schema?

Reply:
Correct. It's likely that we'll need to hang more schema validation off of this in the future but for now this is it.